setup.cfg

```diff
@@ -75,5 +77,6 @@ known_first_party = weldx,weldx_widgets
 sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 
 
-[tool.pytest]
-python_files = *.py
+[tool:pytest]
+# python_files = *.py
```

Comment:
this line wasn't in effect previously (see type in line 80) @marscher
I am not sure what files/test should actually be considered by pytest, only the test folders?

please adjust the configuration as desired (can/should we use `-n 2` here? tests are fast as is but I was just wondering if a specific test setup prevents this in some way?)

Reply:
We can use multiple jobs, but currently I do not think its yet worth the effort.
I split up the test cases in separate files matching the default file pattern of pytest, so this line can be deleted safely.